src/requests/getImages.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if (!query) {
+    return Promise.resolve([]);
+  } else {
+    return axios
+      .get(`https://images-api.nasa.gov/search?q=${query}`)
+      .then((response) => {
+        const imageResults = response.data.collection.items;
+
+        const parseImages = imageResults.filter(
+          (data) => data.data[0].media_type === "image"
+        );
+        const images = parseImages.map((image) => image.links[0].href);
+        return images;
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
From a users point of view it would be better to display an error somewhere they can see it. This is useful for devs but wouldnt make for good user experience